### ISSUE TITLE
Default Username per Group

### ISSUE THREAD

**Issue by LorenzoM604** [keepassxreboot/keepassxc#9582]
## Summary
It would be ideal if KeepassXC allowed me to define a **Default Username** for each group.
This feature currently exists in PasswordSafe.org (which I used prior to KeepassXC.
I keep personal and business passwords in the same database, each in a separate Group/Folder
I have 6 different email addresses.
Having to type a different username when in a branch is time consuming and error prone.
Each entry in a Group, or Sub Group of a Group should inherit the previous Default Username.
A blank Default should inherit the parents default etc..

## Examples
**- Personal**
-- Banking
-- Email
-- Utilities
-- Websites
**- Business 1**
-- Azure
-- AWS
-- Banking
-- Suppliers
**- Business 2**
-- Azure
-- AWS
-- Banking
-- Suppliers
**- Business 3**
-- Azure
-- AWS
-- Banking
-- Suppliers

## Context
I keep personal and business passwords in the same database, each in a separate Group/Folder
I have 6 different email addresses, 1 personal email and 5 business emails.

**Comment by droidmonkey** [comment]
We offer up to 10 of your most frequent usernames. We won't change that behavior to be group-based. That is a very specific feature and it just adds a lot of overhead code for a small population.

We do recommend separating your personal database from your business database. That will solve your problem immediately and offer protection and separation between the two.

**Comment by LorenzoM604** [comment]
Thanks and now that I have finally noticed the drop down, your logic makes sense. I am happy to work with that.

However, it was not obvious from your documentation. I went back through the documentation, and it is not obvious, nor is it mentioned that the username field has a dropdown that includes the 10 most frequent Usernames.

Just a suggestion, the documentation could be modified as below, it would have been obvious (at least to me).
    "We offer up to 10 of your most frequent or default usernames in a dropdown list on the Username field."

Other than that, I'm a happy user and thanks for your time and effort in developing this great product.
